client/my-sites/plugins/jetpack-plugins-setup/index.jsx

```diff
@@ -1,6 +1,6 @@
 import { CompactCard } from '@automattic/components';
 import { localize } from 'i18n-calypso';
-import { filter, get, range } from 'lodash';
+import { filter, get, range, flow } from 'lodash';
```

Comment:
Would be great if we don't introduce a new Lodash usage. Can we just get wrap each call in each other instead of using `flow`?

Reply:
:+1:

Reply:
done.